pkg/transport/http/options.go

```diff
@@ -36,21 +40,23 @@ const (
 
 // EndpointOptions represents the available HTTP options
 type EndpointOptions struct {
-	Method       string
-	Endpoint     string
-	ReadTimeout  time.Duration
-	WriteTimeout time.Duration
-	Codec        string
+	Method        string
+	Endpoint      string
+	ReadTimeout   time.Duration
+	WriteTimeout  time.Duration
+	RequestCodec  string
+	ResponseCodec string
 }
 
 // ParseEndpointOptions parses the given specs options into HTTP options
 func ParseEndpointOptions(options specs.Options) (*EndpointOptions, error) {
 	result := &EndpointOptions{
-		Method:       http.MethodGet,
-		Endpoint:     "/",
-		ReadTimeout:  5 * time.Second,
-		WriteTimeout: 5 * time.Second,
-		Codec:        "json",
+		Method:        http.MethodGet,
+		Endpoint:      "/",
+		ReadTimeout:   5 * time.Second,
```

Comment:
magic numbers :trollface:

Reply:
These are the default read and write timeouts. They could be adjusted or increased in the future if we want to. The could be overriden with the `read_timeout` and `write_timeout` options.

Reply:
I know, the linter would not like that :smile: